parachain/test-parachains/adder/src/wasm_validation.rs

```diff
@@ -24,7 +24,7 @@ use codec::{Encode, Decode};
 
 #[no_mangle]
 pub extern "C" fn validate_block(params: *const u8, len: u32) -> u64 {
-	let params = unsafe { parachain::load_params(params, len) };
+	let params = unsafe { parachain::load_params(params, len as u32) };
```

Comment:
I don't understand the motivation here if `len` is already `u32` per the function definition.

Reply:
oh, I meant to cast it by `usize` 🙈 , I need to devote more attention to this PR.